apps/web/playwright/ab-tests-redirect.e2e.ts

```diff
@@ -2,19 +2,10 @@ import type { Page } from "@playwright/test";
 import { expect } from "@playwright/test";
 
 import { test } from "./lib/fixtures";
-import { testBothFutureAndLegacyRoutes } from "./lib/future-legacy-routes";
 
 test.describe.configure({ mode: "parallel" });
 
-const ensureAppDir = async (page: Page) => {
-  const dataNextJsRouter = await page.evaluate(() =>
-    window.document.documentElement.getAttribute("data-nextjs-router")
-  );
-
-  expect(dataNextJsRouter).toEqual("app");
-};
-
-testBothFutureAndLegacyRoutes.describe("apps/ A/B tests", (routeVariant) => {
+describe("apps/ A/B tests", () => {
```

Comment:
- I am removing all instances of `testBothFutureAndLegacyRoutes`.
- We do not need this because we changed the approach for app router migration. Now, each PR removes the pages router for some pages completely and hence the migrated pages get tested by normal tests